docs/docs/05-database-fundamentals.md

```diff
@@ -71,3 +71,23 @@ RENAME COLUMN old_name TO new_name;
 ```sql
 TRUNCATE TABLE table_name;
 ```
+
+### [1.4] DCL (Data Control Language)
+เป็นกลุ่มคำสั่งที่จะช่วยให้ผู้บริหารฐานข้อมูล (DBA)
+สามารถควบคุมฐานข้อมูลเพื่อ<u>กำหนดสิทธิการอนุญาต</u> (Grant)  หรือ <u>การยกเลิกการเข้าใช้</u> (Revoke) ฐานข้อมูล ซึ่งเป็นกระบวนการป้องกันความปลอดภัยในฐานข้อมูล
+
+* GRANT: คำสั่งนี้<u>ให้สิทธิ์</u> users ในการเข้าถึงฐานข้อมูล
+
+```sql
+GRANT privileges_names ON object TO user;
+```
+
+* REVOKE: คำสั่งนี้จะ<u>ถอนสิทธิ์</u>การเข้าถึงของ users
+
+```sql
+REVOKE privileges_names ON object FROM user;
+```
+
+privileges_names ที่ GRANT หรือ REVOKE ได้
```

Comment:
We can add more description here. For example:

```suggestion
โดย `privileges_names` คือชื่อ privilege ที่เราต้องการ `GRANT` หรือ `REVOKE` ส่วน `object` คือชื่อสิ่งที่เราต้องการจะปรับเปลี่ยนสิทธิ์ของผู้ใช้งานนั้น ๆ เช่น ชื่อตาราง และส่วน `user` คือ ชื่อผู้ใช้งาน
```

Anyway, you can come up with your own wording. 😃

Reply:
done